examples/json/test/data/response.json

```diff
@@ -0,0 +1,4 @@
+{
+  "salutation": "Hello",
```

Comment:
don't need this file, right?

Reply:
good catch (for both), meant to delete. Thanks!